src/Catalog/Extensions/DateTimeExtensions.cs

```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace System
+{
+    public static class DateTimeExtensions
+    {
+        public static DateTime ForceUtc(this DateTime date)
+        {
+            if (date.Kind == DateTimeKind.Unspecified)
```

Comment:
The name may imply there the date will be changed to UTC. This is correct only if the DateTimeKind in Unspecified. What about Local?

Reply:
Fixed in https://github.com/NuGet/NuGet.Services.Metadata/pull/524/commits/b0ab67f19470e9ad3e6e9b6c9e6cd85817e18bcc